Spectrum Update + Questline Extension + Bugfixes (1.1.14)

GW-Dev opened the pull request:
Working on bringing the Spectrum update in the pack up to date with the most recent versions (i.e. the Deeper Down update). Will update the mod and expand the World of Magic questline accordingly. Will also update other mods for best security and compatibility.

Should additionally resolve issue #490 


TODO:
- [x] ensure updates are compatible with the pack and each other
- [x] see if any recipe changes should be made for compatibility
- [x] see if any recipe changes should be made for the cohesion of the experience
- [x] fix #494 
- [x] fix #496 
- [x] fix #467 
- [x] fix #490
- [x] fix #489
- [x] get a handle on the additions to the Spectrum progression
- [x] add holiday stuff
- [x] create additions compat work
- [ ] create the additional quest structure
- [ ] add quest rewards
- [ ] write quest text (oof)

GW-Dev committed:
Mod Updates

GW-Dev committed:
Update Changelog.md

GW-Dev committed:
Add chisel to replicator blacklist

GW-Dev committed:
Fix #467

GW-Dev committed:
Updates (again) + fix #496

GW-Dev committed:
More docs and fixes

Verify fixing of #489, update changelog, modlist, and pack files

GW-Dev committed:
Mod Updates

GW-Dev committed:
Rename AE2-TEXTURE-LICENSE.txt to ae2-texture-license.txt

GW-Dev committed:
Rename MI-TEXTURE-LICENSE.txt to mi-texture-license.txt

GW-Dev committed:
Update (again again)

Had to pin wthit due to config issues is more recent versions

GW-Dev commented:
Time to actually finish this

GW-Dev committed:
Questbook start

Finalize World of Magic page,
start Spectrum: The Deeper Down page

GW-Dev committed:
Questbook Work I

Fix artifacts pictures, begin Deeper Down questline

GW-Dev committed:
packwiz refresh

GW-Dev committed:
Festive lights update

GW-Dev committed:
Updates

GW-Dev committed:
changelog re-wording

GW-Dev committed:
Create Additions integration complete

GW-Dev committed:
remove default gold wire recipe line

GW-Dev committed:
spectrum quest rename

DmitryProskurin commented:
Hello, as i can see, you close #490. If i manually update Spectrum to 1.7.12, that issue become fixed or need something else?

GW-Dev commented:
Yes. I am however planning on finishing this PR this week, so that functionality should be coming to the base pack shortly.

BreadMoirai committed:
Fix bottle_of_failing recipe unlock requirement

progression/unlock_bottle_of_failing
needs to be changed to 
unlocks/items/bottle_of_failing

TheStaticVoid commented:
erm i might have fucked ur pr

GW-Dev commented:
ok, sorry work and school got really busy

GW-Dev committed:
Merge pull request #1 from BreadMoirai/patch-3

Fix bottle_of_failing recipe unlock requirement

GW-Dev committed:
merge

GW-Dev commented:
I think that's everything, there may be a couple things to tweak here and there but yeah

TheStaticVoid commented:
No worries, thank you for your work on this.
